test/scripts/verify.test.js

```diff
@@ -1,55 +1,103 @@
 'use strict'
 require('../setup')
 
+import sinon from 'sinon'
+import axios from 'axios'
+
+import CaptureLogs from '../helpers/captureLogs'
 import { Contracts } from 'zos-lib'
 
-import verify from '../../src/scripts/verify.js';
-import push from '../../src/scripts/push.js';
+import verify from '../../src/scripts/verify.js'
+import push from '../../src/scripts/push.js'
 import ZosPackageFile from '../../src/models/files/ZosPackageFile'
 import ZosNetworkFileFile from '../../src/models/files/ZosNetworkFile'
 
-const ImplV1 = Contracts.getFromLocal('ImplV1');
+const ImplV1 = Contracts.getFromLocal('ImplV1')
 
-contract('verify script', function() {
+contract('verify script', function () {
   const contractAlias = 'Impl'
   const network = 'test'
   const txParams = {}
-  let packageFile;
-  let networkFile;
+  let packageFile
+  let networkFile
+  let logs
+  let axiosStub
```

Comment:
Let's use `this.VARNAME` in the beforeEach blocks instead of `let VARNAME` here, for consistency. This makes sure the tests work if we later abstract them to a behaviour separate from the setup.